stake-pool/single-pool/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "spl-stake-single-pool"
+version = "0.7.0"
+description = "Solana Program Library Single-Stake Pool"
+authors = ["Solana Labs Maintainers <maintainers@solana.foundation>"]
+repository = "https://github.com/solana-labs/solana-program-library"
+license = "Apache-2.0"
+edition = "2021"
+
+[features]
+no-entrypoint = []
+test-sbf = []
+
+[dependencies]
+arrayref = "0.3.6"
+borsh = "0.9"
+mpl-token-metadata =  { version = "1.7.0", features = [ "no-entrypoint" ] }
+num-derive = "0.3"
+num-traits = "0.2"
+num_enum = "0.5.9"
+serde = "1.0.130"
+serde_derive = "1.0.103"
```

Comment:
huh, i actually installed `cargo udeps` to try to catch things to cut, but it missed these... maybe it didnt care because theyre pulled in by other things, so these lines add version constraints but dont actually pull in new packages. it did catch that i didnt need `spl-math`